v2/support/basic_ticket_sample_app/assets/styles.css

```diff
@@ -0,0 +1,3 @@
+body {
+  font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
```

Comment:
@sandlerr I think bootstrap 4 uses that, but I just wanted the basic fallback for simplicity sake.